data/plugins/projectsearch.lua

```diff
@@ -210,7 +204,7 @@ function ResultsView:draw()
       renderer.draw_rect(x, y, w, h, style.line_highlight)
     end
     x = x + style.padding.x
-    local text = string.format("%s at line %d (col %d): ", item.file, item.line, item.col)
+    local text = string.format("%s at line %d (col %d): ", core.root_project():normalize_path(item.file), item.line, item.col)
```

Comment:
I don't think so. Most commands if they refer to a relative path, refer to the root project; so if you did want to generically refer to this somewhere, you'd be using this path, not necessarily the one relative to it in the root project.

Reply:
I'm not sure I follow.
I'm referring to the fact that you can trigger the search from any directory, so if the search started from a directory not in the root project, the resulting filename would look wrong.

Reply:
Well, it would look relative to the root; rather than relative to the folder you initiated the search in (i.e. probably an absolute path). However, given that this is how the CommandView behaves for things like `core:open-file`, I think that's reasonable, no?